site/specs-temp/numbers.json

```diff
@@ -22761,17 +22761,21 @@
             "type": "string"
           },
           "CustomerOrderId": {
+            "description": "Internal customer order id for tracking the order. Only alphanumeric values, dashes and spaces are allowed. Max length is 40 characters.",
```

Comment:
The spec has a place for max length.  Might be better to use that then putting it in the description.